assets/components/common/MobileMenu.vue

```diff
@@ -85,6 +85,15 @@ const sortedContainers = computed(() =>
 const hosts = computed(() => config.hosts.map(({ id, name }) => ({ value: id, label: name })));
 </script>
 <style scoped lang="postcss">
+#side-container {
```

Comment:
I also hate css but ever since i used tailwind it has been better. 

Instead of using id with css, could you check if these are possible with tailwind. I am petty sure the overflow is possible.

Reply:
The overflow as possible, and what I originally did. I just found it a bit weird to have the overflow as a tailwind class, and then attach the calc'ed high property as an id.

I just thought I would wrap them both up in the id.